src/main/resources/embedding-providers-config.yaml

```diff
@@ -239,6 +239,8 @@ stargate:
                   forwarded: Authorization
           properties:
             max-batch-size: 32
+            request-timeout-millis: 15000
```

Comment:
Maybe add a comment to reference the Github issue we have for this change, as background for increasing default timeout?

Reply:
Since we got another similar error report from OpenAI, @maheshrajamani suggests changing the overall default value. What do you think?